lib/helper/Puppeteer.js

```diff
@@ -291,11 +300,13 @@ class Puppeteer extends Helper {
 
   _session() {
     return {
-      start: async () => {
+      start: async (sessionName = '') => {
```

Comment:
I know this was not used yet by Puppeteer but I think it is a good idea if we could pass in config as well.
For instance, in WebDriver we can pass in window size to a new session.
For Puppeteer we also have room for improvement - we can pass in mobile emulation modes, network, etc.

So I propose to have 2 arguments here: session name & config.

Reply:
Thanks for the feedback, I've added the parameters to the Puppeteer start method.